infra/nnfw/python/pack.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# packge
+python3 setup.py sdist bdist_wheel
+
+# deploy for TestPyPI
+twine upload --repository-url https://test.pypi.org/legacy/ dist/*
```

Comment:
I guess `pack.sh` is not for typical user, but for maintainer. Also, `pack.sh` requires `twine` installed.

Reply:
Yes, `pack.sh` is written for maintainer. I will add `sudo apt install twine` on README.md for packaging.